OnServiceChange/handler.ts

```diff
@@ -209,11 +236,28 @@ export const storeDocumentApimToDatabase = (
                 { apimSubscription, apimUser },
                 apimData => mapDataToTableRow(retrievedDocument, apimData),
                 createUpsertSql(config),
-                sql => queryDataTable(pool, sql)
+                sql => queryDataTable(pool, sql),
+                res => {
+                  telemetryClient.trackEvent({
```

Comment:
Consider move this statement in a dedicated functions, so to keep pipeline steps at the same level of abstraction

Reply:
Ok, hope this is what you mean: 79a2765